File: C++/program3.cpp
#include<iostream>
using namespace std;

int main()
{
    int iOptain = 0;
    int iTotal = 0;
    float fPercentage = 0.0f;

    cout<<"Enter Total Number of Marks : ";
    cin>>iTotal;

    cout<<"Enter Optained Number of Marks :";
    cin>>iOptain;

    fPercentage = ((float)iOptain / (float)iTotal) * 100;
    cout<<"Percentage is :" <<+fPercentage;

    return 0;
}

File: JAVA/program3.java
import java.util.*;
class program3
{
    public static void main(String[] A)
    {
        Scanner sobj = new Scanner(System.in);

        int iOptain = 0;
        int iTotal = 0;
        float fPercentage = 0.0f;

        System.out.println("Enter the Optained Marks :");
        iOptain = sobj.nextInt();

        System.out.println("Enter the Total Marks :");
        iTotal = sobj.nextInt();

        fPercentage = ((float)iOptain / (float)iTotal) * 100;
        System.out.println("Percentage is :" + fPercentage);

    }
}